shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -388,6 +388,7 @@ void onStart() {
     Log.v(TAG, "onStart()");
     ensureAlive();
     doInitialFlutterViewRun();
+    flutterView.setVisibility(View.VISIBLE);
```

Comment:
nit: please add some comment(s) here and below about why we're doing this, with a link to the bug

Reply:
Done